### ISSUE TITLE
Add another project to run nominatim in docker

### ISSUE THREAD

**Issue by impocode** [osm-search/Nominatim#3391]
Hello! I developed a project to make it easier to run nominatim on docker based on [nominatim-docker](https://github.com/mediagis/nominatim-docker). Is this solution suitable to add to the ["Basic Installation"](https://nominatim.org/release-docs/develop/admin/Installation/) page?

My project: https://github.com/ready-to-use-docker/nominatim

**Comment by lonvia** [comment]
No. There is no documentation, not even an explanation what it is and no attribution to this project here.

Also, please note that you will have to make a continued commitment to respond to user questions and issues regarding the docker container in order to be listed.
